PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -317,27 +324,33 @@ def nanoAOD_customizeCommon(process):
         nanoAOD_addDeepBTag_switch = cms.untracked.bool(False),
         nanoAOD_addDeepBoostedJet_switch = cms.untracked.bool(True),
         nanoAOD_addDeepDoubleX_switch = cms.untracked.bool(True),
+        nanoAOD_addDeepDoubleXV2_switch = cms.untracked.bool(True),
         nanoAOD_addParticleNet_switch = cms.untracked.bool(True),
         jecPayload = cms.untracked.string('AK8PFPuppi')
         )
-    # deepAK8 should not run on 80X, that contains ak8PFJetsCHS jets
+    # Don't run on old mini due to compatibility
+    # 80X contains ak8PFJetsCHS jets instead of puppi
     run2_miniAOD_80XLegacy.toModify(nanoAOD_addDeepInfoAK8_switch,
                                     nanoAOD_addDeepBTag_switch = True,
                                     nanoAOD_addDeepBoostedJet_switch = False,
                                     nanoAOD_addDeepDoubleX_switch = False,
+                                    nanoAOD_addDeepDoubleXV2_switch = False,
                                     nanoAOD_addParticleNet_switch = False,
                                     jecPayload = 'AK8PFchs')
-    # for Re-MiniAOD: no need to re-run DeepAK8, DeepDoubleX and ParticleNet
-    run2_miniAOD_devel.toModify(
-        nanoAOD_addDeepInfoAK8_switch,
-        nanoAOD_addDeepBoostedJet_switch = False,
-        nanoAOD_addDeepDoubleX_switch = False,
-        nanoAOD_addParticleNet_switch = False,
-        )
+    # Don't rerun where already present
+    for modifier in run2_miniAOD_devel, run2_nanoAOD_106Xv1
+        modifier.toModify(
+            nanoAOD_addDeepInfoAK8_switch,
+            nanoAOD_addDeepBoostedJet_switch = False,
+            nanoAOD_addDeepDoubleX_switch = False,
+            nanoAOD_addDeepDoubleXV2_switch = False,
+            nanoAOD_addParticleNet_switch = False,
+            )
```

Comment:
This is ok for the run2_miniAOD_devel that is going to serve the UL-reminiAOD
but is not ok for the run2_nanoAOD_106Xv1 (already made UL miniAOD) where the ParticleNet and DeepDoubleXV2 do not exist yet.

easiest is to add a separate block for the two things needed

   for modifier in run2_miniAOD_devel, run2_nanoAOD_106Xv1
         modifier.toModify(
             nanoAOD_addDeepInfoAK8_switch,
             nanoAOD_addDeepBoostedJet_switch = False,
             nanoAOD_addDeepDoubleX_switch = False,
             )